src/utils/addressHelpers.ts

```diff
@@ -18,7 +18,7 @@ export const getMulticallAddress = () => {
   return getAddress(addresses.multiCall)
 }
 export const getWbnbAddress = () => {
-  return getAddress(tokens.wbnb.address)
+  return tokens.wbnb.address
```

Comment:
Don't worry about this now but these helpers are not really necessary now.

Reply:
Ah yea good point - I actually thought that when making this change just didn't go back and kill. I'll get rid 👍